src/components/logo.js

```diff
@@ -1,28 +1,77 @@
 import React from "react"
-import { StaticQuery, graphql } from "gatsby"
-import Img from "gatsby-image"
+import styled from "styled-components"
 
-const Logo = ({ className, children }) => (
-  <StaticQuery
-    query={graphql`
-      query {
-        placeholderImage: file(relativePath: { eq: "sandbox-banner.png" }) {
-          childImageSharp {
-            fluid(maxHeight: 40) {
-              ...GatsbyImageSharpFluid_tracedSVG
-            }
-          }
-        }
-      }
-    `}
-    render={data => (
-      <Img
-        className={className}
-        fluid={data.placeholderImage.childImageSharp.fluid}
+const StyledSVG = styled.svg`
+  #Path {
+    transition: fill 0.3s;
+    fill: ${props => props.fillColor};
+  }
+  #Shadow {
+    transition: fill 0.3s;
+    fill: ${props => props.shadowColor};
+  }
+`
+
+const Logo = ({ size = "20px", color = "white" }) => {
+  let fillColor
+  let shadowColor
```

Comment:
I think it's preferable to declare `const shadowColor` and use a ternary operator
```suggestion
  const shadowColor = color === "white" ? "rgb(227, 231, 236)" : "rgb(118, 135, 162)"
```